mongonaut/__init__.py

```diff
@@ -1,6 +1,6 @@
 __author__ = 'Daniel Greenfeld'
 
-VERSION = (0, 2, 21)
+VERSION = (0, 2, 20)
```

Comment:
Version decrease?

Reply:
Dear Garry, Thank you for the quick answer.

I should have missed the latest version.  For some reason I pip installed the django-mongonaut==0.2.20.